### DISCUSSION TITLE
Use SQL in a mutate, when translation is not available

### DISCUSSION THREAD

**Discussion by hershelm** [ibis-project/ibis#10306] [Q&A] [answered]
I would like to use some specific Snowflake JSON extraction functions, which are not yet available in the Snowflake backend. I am wondering if it's possible to insert pure SQL in-line into a `mutate()` command, as you can in R's `dbplyr` (see article below):

https://dbplyr.tidyverse.org/articles/sql.html#what-happens-when-dbplyr-fails

Thanks

**Comment by gforsyth** [comment, marked as answer]
Yep, sure is. You want a scalar udf builtin:
https://ibis-project.org/reference/scalar-udfs.html#ibis.expr.operations.udf.scalar.builtin
